docs/plus/pod.md

```diff
@@ -57,4 +93,267 @@ const chart = new k.Chart(app, 'Chart');
 const pod = new kplus.Pod(chart, 'Pod', {
   serviceAccount: kplus.ServiceAccount.fromServiceAccountName('aws'),
 });
-```
\ No newline at end of file
+```
+
+## Select a pod(s)
+
+Pods can also be selected by various mechanisms. These selections are often used in other
+cdk8s+ API's, such as [pod selection](./pod.md#pod-selection) during scheduling.
+
+### Select pods with labels
+
+Selects all pods that have the `app=store` label.
+
+```ts
+const pods = kplus.Pod.labeled(kplus.LabelQuery.is('app', 'store'));
+```
+
+### Select pods with labels in a particular namespace
+
+Pod selection can also be scoped to specific namespaces.
+This is done using the `.namespaced` method, which can accept any [namespace selector](./namespace.md#select-a-namespaces).
+
+For example, select all pods that have the `app=store` label in the `backoffice` namespace:
+
+```ts
+const pods = kplus.Pod.labeled(kplus.LabelQuery.is('app', 'store'))
+  .namespaced(kplus.Namespace.named('backoffice'));
+```
```

Comment:
What do you think of putting these static methods in a separate `Pods` class to emphasize that a selection could contain any number of pods (zero or more)? I'm wondering if this could possibly match the intent of this line of code a little better: "selecting all `Pods` `labeled` with `app=store` and `namespaced` under `backoffice`".

This would also make some of the other class names a little bit clearer, e.g. `LabeledPod` -> `LabeledPods` makes it more clear that `LabeledPod` isn't just a specific type of pod. :-)

Reply:
Yeah I agree, will change.